docs/UserGuide.adoc

```diff
@@ -483,13 +475,15 @@ To `undo` the `delete` command:
 
 . Type *undo* into the Command Box and press kbd:[Enter]:
 +
-image::UndoAndRedo/undocommand.png[, 550]
+image::UndoAndRedo/undocommand.png[, 500]
 
-. You will see a success message displays in the Result Box, and the transaction (Chicken Rice)
+. You will see a success message displays in the Result Box, and the transaction "Chicken Rice"
 adds back to its original position in the list:
```

Comment:
You might want:
... _returns_ to its original position ...

Reply:
Fixed